addons/mail/static/src/messaging/component/activity/activity.js

```diff
@@ -146,22 +146,7 @@ class Activity extends Component {
      */
     _onClickEdit(ev) {
         ev.preventDefault();
```

Comment:
just wondering why `preventDefault()`? It makes sense for `<a href="#"/>` but here it's on a `<button/>` so it's useless I guess?